game/consumers.py

```
@@ -0,0 +1,74 @@
+            self.room_group_name,
+            self.channel_name
+        )
+
+        await self.accept()
+
+        await self.send(text_data=json.dumps({
+            'message': 'Connected to room',
+            'room_code': self.room_code
+        }))
+
+    async def disconnect(self, close_code):
```

Ruff (0.15.13)

[warning] Unused method argument: `close_code`

(ARG002)